changelogs/fragments/20240613-fix-README-bagdes.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
This should be `trivial`. You can also just skip a changelog fragment for docs changes.

Reply:
should I just remove that for that skip? I just added changelog fragment for [changelog rule](https://github.com/ansible-collections/kubernetes.core/blob/main/.github/workflows/changelog.yaml)

Reply:
Either way is fine. We can attach a skip-changelog label for PRs that don't need a changelog, which will bypass the changelog workflow.